ner_v2/api.py

```diff
@@ -161,11 +161,34 @@ def date(request):
             entity_output = date_detection.detect_bulk(messages=message)
 
         ner_logger.debug('Finished %s : %s ' % (parameters_dict[PARAMETER_ENTITY_NAME], entity_output))
-    except TypeError as e:
-        ner_logger.exception('Exception for date: %s ' % e)
-        return HttpResponse(status=500)
+    except InvalidTextRequest as err:
```

Comment:
Shouldn't one of the except blocks be the original TypeError as well?

Reply:
TypeError and KeyError are handle in single exception in new code

Reply:
I don't think we should catch generic exception anywhere. It is what caused a lot of problems to begin with.